model/trace/database.yaml

```
@@ -232,8 +269,33 @@ groups:
         tag: tech-specific
       - ref: db.collection.name
         brief: The name of the SQL table that the operation is acting upon.
+        examples: ['users', 'dbo.products']
         tag: tech-specific
+      - ref: db.collection.namespace
+        brief: The namespace containing database objects, fully qualified within the server address and port.
+        note: >
+          Namespace contains different identifiers depending on the database system:
+
+          * MySQL - database or schema name. For example, `products.dbo`
+          * PostgreSQL - database and schema name. For example, `products.public`
+          * MS SQL Server - instance name, database and schema name. For example, `test.products.public`
+          * Oracle - instance name and schema name. For example, `test.products`
```

Adding Cosmos DB and MongoDB. One note for MongoDB- "namespace" refers to the fully qualified collection name *including* the collection name itself, but given the definition of this attribute collection name wouldn't be included.

```suggestion
          * Oracle - instance name and schema name. For example, `test.products`
          * Cosmos DB - database name. For example, `products`
          * MongoDB - database name. For example, `products`
```

---

thank you! Reflected in the `db.namespace` brief overrides on Cosmos and Mongo